packages/jats-to-myst/src/transforms/typography.ts

```diff
@@ -0,0 +1,65 @@
+import type { Plugin } from 'unified';
+import type { GenericParent } from 'myst-common';
+import { copyNode } from 'myst-common';
+import { selectAll } from 'unist-util-select';
+import { modifyChildren } from 'unist-util-modify-children';
+import { Tags } from 'jats-xml';
+import type { VFile } from 'vfile';
+
+/**
+ * Lift a child from a parent node. All information on the paragraph is copied (e.g. classes)
+ *
+ * ```
+ * [ ..., {paragraph: [child1, math1, child2, child3, math2]}, ... ]
+ * [ ..., {paragraph: [child1]}, math1, {paragraph: [child2, child3]}, math2, ... ]
+ * ```
+ *
+ * @param tree
+ * @param parentNodeType The parent (e.g. a paragraph)
+ * @param childNodeType The child (e.g. a list or math)
+ */
+function unnestTransform(tree: GenericParent, parentNodeType: string, childNodeType: string) {
```

Comment:
This is something that should eventually be reused in other places, came from myst-transforms.

Reply:
I think we should move this over to myst-transforms and deduplicate, then it should be good to go!